sample_client/src/main/java/com/owncloud/android/lib/sampleclient/MainActivity.java

```diff
@@ -75,7 +74,7 @@ public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.main);
 
-        Timber.plant();
+        Timber.plant(new DebugTree());
```

Comment:
Needed?

Reply:
The complete file is pointless. https://github.com/owncloud/android-library/pull/471

Reply:
rebase error, fix this.